Uno.Cupertino/Styles/Application/ColorPalette.xaml

```diff
@@ -12,12 +12,42 @@
 
 		<!-- Light Theme -->
 		<ResourceDictionary x:Key="Light">
-
+			<Color x:Key="CupertinoBlueColor">#007BFF</Color>
+			<Color x:Key="CupertinoGreenColor">#34C759</Color>
+			<Color x:Key="CupertinoIndigoColor">#5856D6</Color>
+			<Color x:Key="CupertinoOrangeColor">#FF9500</Color>
+			<Color x:Key="CupertinoPinkColor">#FF2D54</Color>
+			<Color x:Key="CupertinoPurpleColor">#AF52DE</Color>
+			<Color x:Key="CupertinoRedColor">#FF3B30</Color>
+			<Color x:Key="CupertinoTealColor">#5AC7FA</Color>
+			<Color x:Key="CupertinoYellowColor">#FFCC00</Color>
+
+			<Color x:Key="CupertinoPrimaryGrayColor">#8E8E93</Color>
+			<Color x:Key="CupertinoSecondaryGrayColor">#AEAEB2</Color>
+			<Color x:Key="CupertinoTertiaryGrayColor">#C7C7CC</Color>
+			<Color x:Key="CupertinoQuaternaryGrayColor">#D1D1D6</Color>
+			<Color x:Key="CupertinoQuinaryGrayColor">#E5E5EA</Color>
+			<Color x:Key="CupertinoSenaryGrayColor">#F2F2F7</Color>
 		</ResourceDictionary>
 
 		<!-- Dark Theme -->
 		<ResourceDictionary x:Key="Dark">
-
+			<Color x:Key="CupertinoBlueColor">#0A84FF</Color>
+			<Color x:Key="CupertinoGreenColor">#30D158</Color>
+			<Color x:Key="CupertinoIndigoColor">#5E5CE6</Color>
+			<Color x:Key="CupertinoOrangeColor">#FF9F0A</Color>
+			<Color x:Key="CupertinoPinkColor">#FF375F</Color>
+			<Color x:Key="CupertinoPurpleColor">#BF5AF2</Color>
+			<Color x:Key="CupertinoRedColor">#FF443A</Color>
+			<Color x:Key="CupertinoTealColor">#64D3FF</Color>
+			<Color x:Key="CupertinoYellowColor">#FFD60A</Color>
+
+			<Color x:Key="CupertinoPrimaryGrayColor">#8E8E93</Color>
+			<Color x:Key="CupertinoSecondaryGrayColor">#636366</Color>
+			<Color x:Key="CupertinoTertiaryGrayColor">#48484A</Color>
+			<Color x:Key="CupertinoQuaternaryGrayColor">#3A3A3C</Color>
+			<Color x:Key="CupertinoQuinaryGrayColor">#2C2C2E</Color>
+			<Color x:Key="CupertinoSenaryGrayColor">#1C1C1E</Color>
```

Comment:
Honestly I like the Tertiary/Quaternary naming style but I'd like to see what Design wants to name them. Either this or something like CupertinoGray/CupertinoGray1/CupertinoGray2/etc. 

I'll ping Martin, I can't find him on GitHub

Reply:
my bad, I did not see your comment in time. I am supposed to meet with him about that.

Reply:
No worries, it's fine to merge it, the PR will stay here and we can document the discussion